metadata.json

```diff
@@ -146,10 +158,12 @@
       "operatingsystem": "Gentoo"
     },
     {
-      "operatingsystem": "Raspbian",
+      "operatingsystem": "windows",
```

Comment:
this needs to be rebased. Please don't drop Raspbian :)

Reply:
hello it should be ok now